folsom/src/main/java/com/spotify/folsom/client/ascii/DefaultAsciiMemcacheClient.java

```diff
@@ -90,6 +95,7 @@ public DefaultAsciiMemcacheClient(
     SetRequest request = SetRequest.casSet(byteKey, valueBytes, ttl, cas);
     CompletionStage<MemcacheStatus> future = rawMemcacheClient.send(request);
     metrics.measureSetFuture(future);
+    tracer.span("folsom.set", future, "set", key);
```

Comment:
If we need to do explicit propagation this would have to look something like...
```java
public CompletionStage<MemcacheStatus> set(String key, V value, int ttl, long cas, Span span) {
...
tracer.span("folsom.set", future, "set", key, span);

```

Reply:
I'd like to avoid polluting the Folsom API if at all possible :)

Reply:
👍 - agreed